.github/workflows/podvm_builder.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
         file: |
           podvm/${{ matrix.dockerfile }}
         build-args: |
-          "GO_VERSION=1.20.7"
+          "GO_VERSION=1.20.8"
```

Comment:
hi @stevenhorsman , for coincidence yesterday I changed that workflow to read the properties from versions.yaml but I kept it closed to send alongside other stuffs. Looking at this change now I realized on how many places some properties are hard-coded! So I think I will expand my changes to include other workflows, what do you think? 

Don't want to block this PR, btw.

Reply:
Yeah, I was surprised how many places go version was still hard-coded, so anything we can do to reduce that is great